doc/api/http2.md

```diff
@@ -24,8 +24,9 @@ be emitted either by client-side code or server-side code.
 
 ### Server-side example
 
-The following illustrates a simple, plain-text HTTP/2 server using the
-Core API:
+The following illustrates a simple HTTP/2 server using the Core API.
+Note the use of [`http2.createSecureServer()`][] since HTTPS is required
+for [most web browsers](https://caniuse.com/#feat=http2).
```

Comment:
No browsers support unencrypted HTTP2 ([HTTP/2 spec FAQ](https://http2.github.io/faq/#does-http2-require-encryption))
You can change this to "since no browsers support unencrypted HTTP2" with the link to HTTP/2 spec FAQ

Ditto for line 1735